cmd/ftl/cmd_new.go

```diff
@@ -58,6 +60,8 @@ func (i newGoCmd) Run(ctx context.Context) error {
 
 	logger := log.FromContext(ctx)
 	logger.Debugf("Creating FTL Go module %q in %s", name, path)
+
+	i.GoVersion = compile.DefaultGoModVersion
```

Comment:
This should never be needed, either pull it from the `go.mod` file or if that's _really_ not possible, from the `go` tool itself.

Reply:
Yeah maybe we should use the go tool for this one too. Before this change the version was hard coded in the template.

Reply:
Ah yes this is for `ftl new` - in that case it should definitely use the Go tool.